scripts/install_idyntree_bindings.m

```diff
@@ -0,0 +1,118 @@
+function install_idyntree_bindings(varargin)
+% authored by Silvio Traversaro
+% modified by Prashanth Ramadoss
+% install iDynTree matlab bindings in the local directory
+% setup a default install path
+% or allow the user to specify path as
+% install_iDynTree('installPrefix', '<path-to-install>')
+p = inputParser;
+default_install_prefix = fullfile(pwd, '../deps/', 'idyntree-matlab');
```

Comment:
I would need to double check, but perhaps using `mfilename` we can remove the requirement for the user to actually be in the `scripts` directory, removing a class of possible problems (such as users adding `scripts` to the matlabpath and running `install_idyntree_bindings` from another directory).

Reply:
Getting the folder name from `mfilename` allows calling the script from a different folder by adding to the path while installing the dependencies in the necessary folder. 
Thank you for this suggestion.

Reply:
Addressed in https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/commit/2295787f336f3825bc6366453dbd792b52198801